sys/net/gnrc/sock/ip/gnrc_sock_ip.c

```diff
@@ -36,7 +36,7 @@ int sock_ip_create(sock_ip_t *sock, const sock_ip_ep_t *local,
         (local->netif != remote->netif)) {
         return -EINVAL;
     }
-    memset(&sock->local, 0, sizeof(sock_ip_t));
+    memset(&sock->local, 0, sizeof(sock_ip_ep_t));
```

Comment:
> And, hmm, that was a quick fix. It compiled, so it must be correct ? ;-)

Nope, tested with `tests/gnrc_sock_ip` on IoT-LAB, but since this and the following variable were initialized with 0 it fell through.